### DISCUSSION TITLE
Is there a way to let the bot play in a VC 24/7? Acting as a radio?

### DISCUSSION THREAD

**Discussion by ayyneya** [jagrosh/MusicBot#1079] [Questions]
Title, and thank you.

**Comment by jagrosh** [comment]
Yes, you will need to create a [local playlist](https://jmusicbot.com/playlists/#local-playlists) and then set that as the [autoplaylist](https://jmusicbot.com/commands/#autoplaylist-playlistname). If you want this to begin playing as soon as the bot starts up, you'll also need to specify [a voice channel](https://jmusicbot.com/commands/#setvc-channel).
